### PR TITLE
Add support for Swift 3.

### PR DESCRIPTION
**Pull request by pedrovgs**
Update the project code to the new Swift 3 guidelines and update dependencies and project configuration to support XCode 8 and Swift 3.

### CODE REVIEW

**Comment by pedrovgs** [comment]
As Travis-CI can't download SwiftCheck version using Swift 3 we are going to leave this PR as open until SwiftCheck 7.0 be released.

File: MaxibonKataIOS/Chat.swift [hunk 1]
```diff
@@ -10,6 +10,6 @@ import Foundation
 
 public protocol Chat {
 
-    func sendMessage(message: String)
+    func sendMessage(_ message: String)
```
Comment (davideme) [inline MaxibonKataIOS/Chat.swift]: prefer the new guidelines, and use, send(message: String)

File: MaxibonKataIOS/Slack.swift [hunk 2]
```diff
@@ -10,9 +10,9 @@ import Foundation
 
 struct Slack: Chat {
 
-    func sendMessage(message: String) {
+    func sendMessage(_ message: String) {
```
Comment (davideme) [inline MaxibonKataIOS/Slack.swift]: prefer send(message

File: MaxibonKataIOSTests/MockSlack.swift [hunk 3]
```diff
@@ -13,7 +13,7 @@ class MockChat: Chat {
 
     var messageSent: String? = nil
 
-    func sendMessage(message: String) {
+    func sendMessage(_ message: String) {
```
Comment (davideme) [inline MaxibonKataIOSTests/MockSlack.swift]: same here

**Comment by Serchinastico** [comment]
I updated the code with your proposed changes and changed the `SwiftCheck` dependency so that it loads version `0.7.1` which has been released.

**Pull request merged**